grakn-engine/src/main/java/ai/grakn/engine/controller/VisualiserController.java

```diff
@@ -240,8 +241,9 @@ private String compute(Request req, Response res) {
     @ApiOperation(value = "Pre materialise all the rules on the graph.")
     @ApiImplicitParam(name = "keyspace", value = "Name of graph to use", dataType = "string", paramType = "query")
     private String preMaterialiseAll(Request req, Response res) {
-        try (GraknGraph graph = getInstance().getGraph(getKeyspace(req))) {
-            Reasoner.precomputeInferences(graph);
+        try (EngineGraknGraph graph = getInstance().getGraph(getKeyspace(req))) {
+            //TODO: Fix ugly casting here
```

Comment:
This is because reasoner is expecting a GraknGraph and the factory now produces a EngineGraknGraph and they are different. Need to fix that yes. Just not sure how at the moment.